script/staging/FullStagingDeploy.s.sol

```diff
@@ -306,6 +293,26 @@ contract FullStagingDeploy is Script {
             name:       "usdeBurnData",
             decimals:   18
         });
+        MainnetControllerInit.setRateLimitData({
+            key:        RateLimitHelpers.makeAssetKey(
+                            MainnetController(instance.controller).LIMIT_4626_DEPOSIT(),
```

Comment:
Reformatted for easier review:

```solidity
        bytes32 mintKey = MainnetController(instance.controller).LIMIT_USDE_MINT();
        bytes32 burnKey = MainnetController(instance.controller).LIMIT_USDE_BURN();
    
        bytes32 susdsDepositKey = RateLimitHelpers.makeAssetKey(
            MainnetController(instance.controller).LIMIT_4626_DEPOSIT(),
            address(MainnetController(instance.controller).susde())
        );

        bytes32 susdsWithdrawKey = RateLimitHelpers.makeAssetKey(
            MainnetController(instance.controller).LIMIT_4626_WITHDRAW(),
            address(MainnetController(instance.controller).susde())
        );

        // Extra rate limit configuration
        MainnetControllerInit.setRateLimitData(mintKey,     IRateLimits(instance.rateLimits), rateLimitData6,  "usdeMintData",      6);
        MainnetControllerInit.setRateLimitData(burnKey,     IRateLimits(instance.rateLimits), rateLimitData18, "usdeBurnData",      18);
        MainnetControllerInit.setRateLimitData(depositKey,  IRateLimits(instance.rateLimits), rateLimitData18, "susdsDepositData",  18);
        MainnetControllerInit.setRateLimitData(withdrawKey, IRateLimits(instance.rateLimits), ateLimitData18,  "susdsWithdrawData", 18);
```

was able to spot duplicated susdsMintData name after reformatting, updated to deposit/withdraw.